qiskit/quantum_info/synthesis/clifford_decompose.py

```diff
@@ -150,10 +155,10 @@ def decompose_clifford_ag(clifford):
         _set_row_z_zero(clifford_cpy, circuit, i)
 
     for i in range(clifford.num_qubits):
-        if clifford_cpy.destabilizer.phase[i]:
+        if clifford_cpy.paulis.phase[i]:
```

Comment:
maybe you can use here destabilizer?

Reply:
Thanks. I didn't make methods for phase. Can I make `stabilizer_phase` and `destabilizer_phase`?

Reply:
yes, I think it will be useful and make the code more readable